scripts/scil_resample_tractogram.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+""" Dumb script to resample a tractogram to a set number of streamlines.
+Might be useful to build training sets for machine learning algorithms, to
+upsample under-represented bundles or downsample over-represented bundles.
+
+Works by either selecting a subset of streamlines or by generating new
+streamlines by adding gaussian noise to existing ones.
+
+It is recommended to use "scil_smooth_streamlines" afterwards to clean the
+noisy new streamlines.
```

Comment:
Please provide your example 3 line execution as example, i.e. resample, smooth, visualize as in the PR conversation. This is really helpful for the future user